lua/framework/console/console.h

```diff
@@ -24,36 +24,35 @@ extern "C"
 /**IDK why this is needed. Seems like counter doesnt work properly without it*/
 #define ConsoleJoinWordsB(x, y) ConsoleJoinWordsA(x, y)
 
-/**Macro for registering a lua_CFunction within the Console
- * singleton, with the function being in the global namespace. Example:
+/**
+ * Macro for registering a `lua_CFunction` within the global namespace.
```

Comment:
is comment correct?

Reply:
`lua_CFunction` is a lua type, see [here](https://www.lua.org/pil/26.1.html). It is has this prototype:
```
int (*lua_CFunction) (lua_State *L);
```
so I think that's what this refers to. The so-called wrapper functions do not have this prototype.